backend/client/deepchecks_client/core/client.py

```diff
@@ -508,17 +507,56 @@ def add_alert_rule(
             greater_than=greater_than
         )
 
+    def add_alert_webhook(self, name: str, https_url: str, http_method: str, description: str = '',
+                          http_headers: t.Dict[str, str] = None, notification_levels: t.List[str] = None) -> int:
```

Comment:
please, use `Optional` in places like here

` http_headers: Dict[str, str] = None` is incorrect, it should be ` http_headers: Optional[Dict[str, str]] = None`

[python doc - typing.Optional](https://docs.python.org/3/library/typing.html#typing.Optional)

if I am not wrong, only mypy static type analyzer allows that and considers that to be fine, but that is not a python standart and my static type analyzer complaints about that

Reply:
you missed 

```
def add_alert_webhook(...http_headers: t.Dict[str, str] = None, notification_levels: t.List[str] = None ...) -> int:
```

Reply:
Fixed